src/components/hub/ChatList.js

```diff
@@ -45,27 +45,30 @@ export default function ChatList() {
       return <p className="chat-list__empty">No chats are available.</p>;
     }
 
-    return filteredConversations.map((obj) => (
-      <ConversationItem
-        key={obj._id}
-        isSelected={activeConv === obj._id}
-        onClickFunc={() => {
-          dispatch(setSelectedConversation({ id: obj._id }));
-          navigateTo(`/#${obj._id}`);
-        }}
-        chatName={
-          obj.name ||
-          getUserFullName(
-            participants[
-              obj[obj.owner_id === currentUser._id ? "opponent_id" : "owner_id"]
-            ] || {}
-          )
-        }
-        chatObject={obj}
-        currentUserId={currentUser._id}
-      />
-    ));
-  }, [conversations, participants, activeConv, currentUser]);
+    return filteredConversations.map((obj) => {
```

Comment:
Also, 

this code

```
 const filteredConversations = conversations.filter(
      (obj) => obj.type === "g" || obj.last_message
    );
```

I recommend to do it as apart of selector, so just create a new selector in `src/store/values/Conversations.js` and use it here instead of `selectAllConversations`